build.gradle

```diff
@@ -581,7 +581,7 @@ def createCheckTypeTask(projectName, taskName, checker, args = []) {
         outputs.upToDateWhen { false }
         source = project("${projectName}").sourceSets.main.java
         classpath = files(project("${projectName}").compileJava.classpath,project(':checker-qual').sourceSets.main.output)
-        destinationDirectory = file("${buildDir}")
+        destinationDirectory = file("${projectDir}/build")
```

Comment:
This is not the recommended fix suggested in https://docs.gradle.org/current/userguide/upgrading_version_8.html#project_builddir 
Wouldn't it be nicer to use the suggested replacement?